docs/user-guide.md

```diff
@@ -44,8 +44,8 @@ laid out on this page. This includes making use of a template model class, which
 consistency with respect to the code structure, but avoids imposing unnecessary restrictions. At present, this class
 must be present in your codebase but the intention is to make it available for import as an external library.
 
-The model code is uploaded as a <b>zip file</b> on the upload files section on the user interface upload form. The
-uploaded code must contain:
+The model code is uploaded as a <b>zip file</b> (however, other file types may be added as options at a later date) on
```

Comment:
Don't include information about _future_ features in documentation, it defines the _current_ state of the project.  When other file types are added, we can add it to the documentation.  Otherwise we'll be adding a lot of 'in the futures' everywhere!

Reply:
I've updated it to add docker container as we're now supporting that.